.travis/run-linters.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-docker pull hadolint/hadolint tag: v1.3.0
+docker pull hadolint/hadolint: v1.3.0
```

Comment:
It is still pulling latest!

https://travis-ci.org/wmde/wikibase-docker/jobs/421565364#L392